test/libsolidity/semanticTests/optimizer/overwritten_assignment_multireturn_function.sol

```diff
@@ -0,0 +1,18 @@
+// This tests that the legacy optimiser
+// assigns the value of the first function
+// output to the variable on the LHS of
+// an inline assembly multi variable assignment
+// that contains the same variable more
+// than once
+contract C {
+  function f() public returns (uint o) {
+    assembly {
+      function g() -> a, b { a := 1 b := 2 }
+      o, o := g()
```

Comment:
This test needs to be updated after https://github.com/ethereum/solidity/pull/9321, right?

Reply:
Yeah, in fact this is true for other tests in this PR as well (https://github.com/ethereum/solidity/pull/9302#discussion_r450685378)